sql/table.go

```diff
@@ -43,6 +43,10 @@ func TestDisableTableLeases() func() {
 	}
 }
 
+func tableDoesntExistUErr(name string) *roachpb.Error {
+	return roachpb.NewUErrorf("table %q does not exist", name)
```

Comment:
can't use `fmt.Errorf` here and convert upstream (not necessary, just nice to push the `*Error` plague up as high as possible)